test-suites/_pool_core_erc721_liquidation.spec.ts

```diff
@@ -346,4 +347,46 @@ describe("ERC-721 Liquidation", () => {
       feeAmount
     );
   });
+
+  it("TC-erc721-liquidation-09 If supplied WETH is not in collateral, after liquidation it should be collateralize", async () => {
+    const {
+      users: [borrower, liquidator],
+      pool,
+      bayc,
+      weth,
+      dai,
+      protocolDataProvider,
+    } = testEnv;
```

Comment:
Can we use fixture rather than the `beforeEach` hook to do the setup? It's more clear and will reduce the time cost.

Reply:
done